src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -598,6 +598,10 @@ src: ["my-package/SomeFont.ttf"]
 
 </Tabs>
 
+:::caution
+We recommend not to store fonts in [`/public`](/en/reference/configuration-reference/#publicdir) as this will result in duplicated files in your build output. Instead, store them in [`/src`](/en/reference/configuration-reference/#srcdir) for example.
```

Comment:
```suggestion
We recommend not putting your font files in [the `public/` directory](/en/reference/configuration-reference/#publicdir). Since Astro will copy these files into that folder at build time, this will result in duplicated files in your build output. Instead, store them somewhere else in your project, such as in [`src/`](/en/reference/configuration-reference/#srcdir).
```
I made up a reason, because I think having extra context is helpful here. Can you make this accurate?